frontend/src/App.tsx

```diff
@@ -455,14 +456,22 @@ class App extends PureComponent<Props, State> {
    */
   clearOldElements = (elements: any, reportId: string): BlockElement => {
     return elements
-      .map((element: any) => {
-        if (element instanceof List) {
-          const clearedElements = this.clearOldElements(element, reportId)
+      .map((elementWrapper: ElementWrapper) => {
+        const simpleElement = elementWrapper.get("element")
+
+        if (simpleElement instanceof List) {
+          const clearedElements = this.clearOldElements(
+            simpleElement,
+            reportId
+          )
           return clearedElements.size > 0 ? clearedElements : null
         }
-        return element.get("reportId") === reportId ? element : null
+
+        return elementWrapper.get("reportId") === reportId
+          ? elementWrapper
+          : null
       })
-      .filter((element: any) => element !== null)
+      .filter((element: any) => element !== null || element)
```

Comment:
Why did you add the `|| element` to the end? Unless I'm misreading this, the final `|| element` breaks the filter (i.e. it will never filter anything out)